deployment/live/gcp/conformance/README.md

```diff
@@ -2,15 +2,23 @@
 
 ## Prerequisites
 
-You'll need to have already configured/created a KMS key which can safely be used by the
-conformance log.
+You'll need to have already configured/created whatever service accounts + IAM permissions
+you require, and update the terragrunt files here to match.
 
-> [Warning]
-> This key should not be used elsewhere or be in any way valuable!
+## Overview
+
+This config uses the [gcp/conformance](/deployment/gcp/conformance) module to define a
```

Comment:
Would the link be `/deployment/module/gcp/conformance`?

Reply:
good spot, ta!